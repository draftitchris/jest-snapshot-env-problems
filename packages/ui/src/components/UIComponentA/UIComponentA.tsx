/** @jsx jsx */
import { jsx, css } from "@emotion/core";
import * as React from "react";

interface IUIComponentAProps {}

export const UIComponentA: React.FunctionComponent<
  IUIComponentAProps
> = ({}) => {
  return (
    <div
      css={css`
        background: red;
      `}
    >
      This is Component A
    </div>
  );
};

UIComponentA.displayName = "UIComponentA";
