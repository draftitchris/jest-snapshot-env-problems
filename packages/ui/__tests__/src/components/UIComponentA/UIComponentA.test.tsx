import * as React from "react";
import { render } from "@testing-library/react";
import * as Serializer from "jest-emotion";
import { matchers } from "jest-emotion";
import { UIComponentA } from "../../../../src/components/UIComponentA/UIComponentA";

expect.extend(matchers);
expect.addSnapshotSerializer(Serializer);

describe("description", () => {
  it("snapshots", () => {
    const { asFragment } = render(<UIComponentA />);
    expect(asFragment()).toMatchSnapshot();
  });
});
