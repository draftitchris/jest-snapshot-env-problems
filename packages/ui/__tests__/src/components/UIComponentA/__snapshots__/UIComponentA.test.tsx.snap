// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`description snapshots 1`] = `
<DocumentFragment>
  <div
    class="emotion-0"
  >
    This is Component A
  </div>
</DocumentFragment>
`;
